sw/device/lib/dif/dif_kmac_unittest.cc

```diff
@@ -8,6 +8,9 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
+#include "kmac_regs.h"
```

Comment:
Nit: should have comment '// Generated', and place them at the end of include pile, like we do in GPIO:
```
// Copyright lowRISC contributors.                                                                                                                                            
// Licensed under the Apache License, Version 2.0, see LICENSE for details.                                                                                                   
// SPDX-License-Identifier: Apache-2.0                                                                                                                                        

#include "sw/device/lib/dif/dif_gpio.h"

#include <limits>

#include "gtest/gtest.h"
#include "sw/device/lib/base/mmio.h"
#include "sw/device/lib/base/testing/mock_mmio.h"

#include "gpio_regs.h"  // Generated
```